tools/src/h5diff/h5diff_common.c

```diff
@@ -56,6 +56,28 @@ static struct h5_long_options l_opts[] = {{"help", no_arg, 'h'},
                                           {"vfd-info-2", require_arg, 'Z'},
                                           {NULL, 0, '\0'}};
 
+static H5FD_onion_fapl_info_t onion_fa_g_1 = {
```

Comment:
I've probably mentioned this before, but definitely the sooner we can move the VFD-specific info out of the tools, the better maintenance-wise.